detox/src/devices/DeviceRegistry.js

```diff
@@ -44,10 +44,14 @@ class DeviceRegistry {
     });
   }
 
-  isDeviceBusy(deviceId) {
+  includes(deviceId) {
     return this._lockfile.read().includes(deviceId);
   }
 
+  getRegisteredDevices() {
```

Comment:
An optimization in case `include` needs to be checked repeatedly.

Reply:
Although, it makes me wonder if we need at all the `includes` method after this change...? 🤔